xmtp_mls/src/groups/subscriptions.rs

```diff
@@ -137,40 +141,6 @@ pub(crate) mod tests {
 
     use futures::StreamExt;
 
-    #[rstest::rstest]
-    #[xmtp_common::test]
-    #[timeout(Duration::from_secs(10))]
-    async fn test_decode_group_message_bytes() {
```

Comment:
this test is no longer relevant, since byte decoding of messages is left up to api_d14n and its extractors